website/src/_posts/guide-to-call-site-generic-variance.md

```diff
@@ -0,0 +1,163 @@
+---
+title: "A guide to call-site generic variance"
+date: 2023-09-18
+tags: guides
+---
+PHPStan has supported what is called declaration-site variance for a long time. An example you might be familiar with is the infamous [`@template-covariant`](/blog/whats-up-with-template-covariant). And although not as often useful, it also has [a contravariant counterpart](https://jiripudil.cz/blog/contravariant-template-types).
```

Comment:
That last link is kind of a shameless plug 😅 I thought the feature deserves a mention. I'm ok with cross-posting relevant parts of that post on PHPStan blog if you want to have better control over it :)

Reply:
It looks like `@template-contravariant` is already supported, but it's missing here: https://phpstan.org/writing-php-code/phpdocs-basics#generics - feel free to add it :)

Reply:
Otherwise I'm okay with this cross-linking :)